os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -265,16 +373,24 @@ static void unwind_backtrace_with_fp(arm_regs_t *regs, struct tcb_s *task)
 		uint32_t current_addr = stack_frame.programCounter;
 		if (unwind_frame_with_fp(&stack_frame, ustacksize) >= 0) {
 			/* Print the call stack address */
+#ifdef CONFIG_FS_ROMFS
+			char buffer[64];
+			get_symbol(current_addr, buffer);
+			lldbg("[<0x%p>] %s\n", (void *)current_addr, buffer);
```

Comment:
When opening of system.map is failed, this can show wrong information due to there is no checking result of get_symbol.

Reply:
Ok.  I have handled it as  follows
1) If get_symbol return fails, Only address will be printed and symbol name will not be printed.
2) If get_symbol is success, It prints both address and corresponding symbol name.